plugins/csv/src/lib.rs

```diff
@@ -79,3 +79,50 @@ impl Default for Config {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+
+    use crate::{Config, CsvPlugin};
+    use alumet::{
+        agent::{
+            self,
+            plugin::{PluginInfo, PluginSet},
+        },
+        pipeline,
+        plugin::PluginMetadata,
+    };
+    use std::time::Duration;
+
+    fn config_to_toml_table(config: &Config) -> toml::Table {
+        toml::Value::try_from(config).unwrap().as_table().unwrap().clone()
+    }
+
+    #[test]
+    fn start_stop() {
+        let default_config = Config { ..Default::default() };
```

Comment:
What is this test testing? The integration test is good enough: it already does start and stop.